core/src/main/java/io/grpc/internal/InternalSubchannel.java

```diff
@@ -542,6 +547,9 @@ private class TransportListener implements ManagedClientTransport.Listener {
     @Override
     public void transportReady() {
       channelLogger.log(ChannelLogLevel.INFO, "READY");
+      for (ClientTransportFilter filter : transportFilters) {
+        filter.transportReady();
```

Comment:
If we were to pass the attributes, they can be gotten via `transport.getAttributes()`.